build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# parameters $1 is build number
+
+export WORKSPACE=`pwd`
+export RPM_VERSION="0.8"
+export BUILD_NUMBER=$1
+
+error() {
+  local parent_lineno="$1"
+  local message="$2"
+  local code="${3:-1}"
+  if [[ -n "$message" ]] ; then
+    echo "Error on or near line ${parent_lineno}: ${message}; exiting with status ${code}"
+  else
+    echo "Error on or near line ${parent_lineno}; exiting with status ${code}"
+  fi
+  exit "${code}"
+}
+
+#build and run unit tests
+trap 'error ${LINENO}' ERR
+virtualenv venv
+source venv/bin/activate
```

Comment:
The reason I asked is because if I run this exact commands on my mac the version of python in venv is 2.7. But if this is not meant for it, then it is alright.